utils/array_utils.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+func StringFoundInArray(item string, arr []string) bool {
```

Comment:
This can be removed since we can use [slices.Contains](https://pkg.go.dev/slices#Contains) instead